packages/odyssey-react-mui/src/Tabs.tsx

```diff
@@ -99,6 +101,7 @@ const Tabs = ({ ariaLabel, initialValue, tabs, value }: TabsProps) => {
       </MuiTabList>
       {tabs.map((tab, index) => (
         <MuiTabPanel
+          data-se={tab.testId}
```

Comment:
Unsure if applying the `testId` to both the tab itself and the content panel is the right thing to do here. Thoughts?

Reply:
Just the tab is fine.